fineract-core/src/main/java/org/apache/fineract/infrastructure/clientblockingreasons/exception/BlockingReasonExceptionNotFoundException.java

```diff
@@ -23,16 +23,16 @@
 public class BlockingReasonExceptionNotFoundException extends AbstractPlatformResourceNotFoundException {
 
     public BlockingReasonExceptionNotFoundException(final Long id) {
-        super("error.msg.id.does.not.exist", "Blocking Reason Settings with identifier " + id + " does not exist", id);
+        super("errors.error.msg.id.does.not.exist", "Blocking Reason Settings with identifier " + id + " does not exist", id);
     }
 
     public BlockingReasonExceptionNotFoundException(final Integer priority, final String level) {
-        super("error.msg.priority.is.already.assigned.to.client.level",
+        super("errors.error.msg.priority.is.already.assigned.to.client.level",
                 "Priority  [" + priority + "] is already assigned to client level " + level, priority, level);
     }
 
     public BlockingReasonExceptionNotFoundException(final String reason, final String level) {
-        super("error.msg.block.reason.is.already.assigned.to.level",
+        super("errors.error.msg.block.reason.is.already.assigned.to.level",
```

Comment:
@hero78 I think you maintain the error code as "error.msg
I have already handled the addition of "errors" on the it on the UI alerts in one my fixes